### PR TITLE
Update pd-ctl docs

### PR DESCRIPTION
**Pull request by rleungx**
### What is changed, added or deleted? (Required)

This PR updates the docs of pd-ctl.

@rleungx will apply version-specific changes in the cherry-picked PR in release-4.0.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
@rleungx Should the Chinese version be updated as well? Is it only applicable to `master`?

**Comment by rleungx** [comment]
PTAL @nolouch @lhy1024

File: pd-control.md [hunk 1]
```diff
@@ -272,47 +277,25 @@ The configuration above is global. You can also tune the configuration by config
...
-```
+- `enable-location-replacement` is used to enable the isolation level check. When you set it to `false`, PD does not improve the isolation level of Region replicas by scheduling.
 
-If you want to use global configuration only for a certain configuration of the namespace, use the following command:
+- `enable-debug-metrics` is used to enable the debug metrics. When you set it to `true`, PD will open some metrics, such as balance-tolerant-size and op influence.
 
-```bash
->> config delete namespace region-schedule-limit ts2 // Delete the region-schedule-limit configuration of the namespace named ts2
-```
+- `store-limit-mode` has two mode for setting limit: auto or manual, an auto-set value can be overwritten by a manual-set value, otherwise it is forbidden (experimental).
```
Comment (lhy1024) [inline pd-control.md:299]: Do we need to introduce how to use `placement-rules`?

File: pd-control.md [hunk 2]
```diff
@@ -388,9 +371,9 @@ Success!
 ......
```
Comment (lhy1024) [inline pd-control.md:371]: member output
```
>> member                               // Display the information of all members
{
  "header":{......},
  "members": [......],
  "leader": {......},
  "etcd_leader": {......},
}
```

**Comment by TomShawn** [comment]
@lhy1024 PTAL again
@rleungx Please resolve the conflict.

File: pd-control.md [hunk 3]
```diff
@@ -307,9 +311,12 @@ Usage:
...
 - `enable-placement-rules` is used to enable placement rules.
 
-### `config placement-rules [disable | enable | load | save | show]`
+- `store-limit-mode` is used to control the mode of the store speed limit mechanism. There are two modes: `auto` and `manual`. In `auto` mode, the stores are automatically ba
+lanced according to load (experimental).
 
-Use this command to configure Placement Rules. For details, refer to [Placement Rules Usage Document](/configure-placement-rules.md).
+#### `config placement-rules [disable | enable | load | save | show]`
+
+The usage can be found in [Configure rules](configure-placement-rules.md#Configure-rules)
```
Comment (TomShawn) [inline pd-control.md:319]:
```suggestion
The usage can be found in [Configure rules](/configure-placement-rules.md#configure-rules)
```

**Comment by yikeke** [comment]
PTAL again @TomShawn

**Comment by yikeke** [comment]
PTAL again @TomShawn

File: pd-control.md [hunk 4]
```diff
@@ -664,50 +684,106 @@ Usage:
...
->> store limit 1 5 region-add          // Set the speed limit of adding peers in store 1 to 5 per minute
->> store limit 1 5 region-remove       // Set the speed limit of deleting peers in store 1 to 5 per minute
->> store limit all 5 region-remove     // Set the speed limit for all stores to delete peers to 5 per minute
->> store limit-scene                   // Display the maximum speed of adding/deleting peers in different loads (only valid when `store-limit-mode` is `auto`)
+>> store label 1 zone cn               // Set the value of the label with the "zone" key to "cn" for the store with the store id of 1
+>> store weight 1 5 10                 // Set the leader weight to 5 and region weight to 10 for the store with the store id of 1
+>> store remove-tombstone              // Remove stores that are in tombstone state
+>> store limit                         // Show limits of adding peer and removing peer operation for all stores
+>> store limit add-peer                // Show limits of adding peer operation for all stores
+>> store limit remove-peer             // Show limits of removing peer operation for all stores
```
Comment (TomShawn) [inline pd-control.md:767]:
```suggestion
>> store limit remove-peer             // Show the limit of removing-peer operations per minute in all stores
```

File: pd-control.md [hunk 5]
```diff
@@ -664,50 +684,106 @@ Usage:
...
+>> store limit all 5                   // Limit 5 adding peer operations and 5 remove peer operations per minute for all stores
```
Comment (TomShawn) [inline pd-control.md:768]:
```suggestion
>> store limit all 5                   // Set the limit of adding-peer operations to 5 and the limit of removing-peer operations to 5 per minute for all stores
```

File: pd-control.md [hunk 6]
```diff
@@ -664,50 +684,106 @@ Usage:
...
+>> store limit 1 5                     // Limit 5 adding peer operations and 5 remove peer operations per minute for store 1
```
Comment (TomShawn) [inline pd-control.md:769]:
```suggestion
>> store limit 1 5                     // Set the limit of adding-peer operations to 5 and the limit of removing-peer operations to 5 per minute for store 1
```

File: pd-control.md [hunk 7]
```diff
@@ -664,50 +684,106 @@ Usage:
...
+>> store limit all 5 add-peer          // Limit 5 adding peer operations per minute for all stores
```
Comment (TomShawn) [inline pd-control.md:770]:
```suggestion
>> store limit all 5 add-peer          // Set the limit of adding-peer operations to 5 per minute for all stores
```

File: pd-control.md [hunk 8]
```diff
@@ -664,50 +684,106 @@ Usage:
...
+>> store limit 1 5 add-peer            // Limit 5 adding peer operations per minute for store 1
```
Comment (TomShawn) [inline pd-control.md:771]:
```suggestion
>> store limit 1 5 add-peer            // Set the limit of adding-peer operations to 5 per minute for store 1
```

File: pd-control.md [hunk 9]
```diff
@@ -664,50 +684,106 @@ Usage:
...
+>> store limit 1 5 remove-peer         // Limit 5 removing peer operations per minute for store 1
```
Comment (TomShawn) [inline pd-control.md:772]:
```suggestion
>> store limit 1 5 remove-peer         // Set the limit of removing-peer operations to 5 per minute for store 1
```

File: pd-control.md [hunk 10]
```diff
@@ -664,50 +684,106 @@ Usage:
...
+>> store limit all 5 remove-peer       // Limit 5 removing peer operations per minute for all stores
```
Comment (TomShawn) [inline pd-control.md:773]:
```suggestion
>> store limit all 5 remove-peer       // Set the limit of removing-peer operations to 5 per minute for all stores
```

File: pd-control.md [hunk 11]
```diff
@@ -664,50 +684,106 @@ Usage:
...
+>> store limit-scene                   // Show all limit scene (experimental)
```
Comment (TomShawn) [inline pd-control.md:774]:
```suggestion
>> store limit-scene                   // Show all limit scenarios (experimental)
```

File: pd-control.md [hunk 12]
```diff
@@ -664,50 +684,106 @@ Usage:
...
 {
   "Idle": 100,
   "Low": 50,
   "Normal": 32,
   "High": 12
 }
->> store limit-scene idle 100          // In the scenario of setting load to idle, set the speed limit of adding/deleting peers to 100 per minute
+>> store limit-scene idle 100 // set rate to 100 in the idle scene (experimental)
```
Comment (TomShawn) [inline pd-control.md:781]: The original version is better?
```suggestion
>> store limit-scene idle 100 // In the scenario of setting load to idle, set the speed limit of adding/deleting peers to 100 per minute (experimental)
```

File: pd-control.md [hunk 13]
```diff
@@ -664,50 +684,106 @@ Usage:
...
 ```
 
-> **Note:**
+> **Notice**
```
Comment (TomShawn) [inline pd-control.md:784]:
```suggestion
> **Note:**
```

File: pd-control.md [hunk 14]
```diff
@@ -664,50 +684,106 @@ Usage:
...
 >
-> The effectiveness of the store limit depends on the order in which the commands are executed. For example, if you first execute `store limit 1 5 region-add` which sets the speed limit of adding peers in store 1 to 5 per minute, and then execute `store limit 1 10 region-add`. At this time, the speed limit for adding peers in store 1 is modified to 10 per minute; otherwise, if you first execute `store limit 1 10 region-add` which sets the speed limit of adding peers in all stores to 10 per minute, and then execute `store limit 1 5 region-add`. At this time, only the speed limit of adding peers in store 1 is modified to 5 per minute.
+> When using `store limit` command, the original `region-add` and `region-remove` are deprecated, please use `add-peer` and `remove-peer`.
```
Comment (TomShawn) [inline pd-control.md:786]:
```suggestion
> When you use the `store limit` command, the original `region-add` and `region-remove` are deprecated. Use `add-peer` and `remove-peer` instead.
```

File: pd-control.md (hunk 12 above)
Comment (rleungx) [reply pd-control.md:780]: it is a little different from the original meaning.

File: pd-control.md [hunk 15]
```diff
@@ -664,50 +684,106 @@ Usage:
...
+    >> scheduler config balance-hot-region-scheduler set min-hot-byte-rate 100
+    ```
+
+- `min-hot-key-rate` means the smallest key counted, it is usually 10
+
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set min-hot-key-rate 10
+    ```
+
+- `max-zombie-rounds` means the maximum number of heartbeat thatan operator is considered as a pending influence.
```
[the remark this answers is not in the document]
Comment (rleungx) [reply pd-control.md:721]: @lhy1024 PTAL

File: pd-control.md [hunk 16]
```diff
@@ -664,50 +683,106 @@ Usage:
...
+    >> scheduler config balance-hot-region-scheduler set min-hot-byte-rate 100
+    ```
+
+- `min-hot-key-rate` means the smallest key counted, which is usually 10.
+
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set min-hot-key-rate 10
+    ```
+
+- `max-zombie-rounds` means the maximum number of heartbeat thatan operator is considered as a pending influence.
```
Comment (lhy1024) [inline pd-control.md:721]:
```suggestion
- `max-zombie-rounds` means the maximum number of heartbeat that an operator is considered as a pending influence. If it is adjusted larger, more operators may be included in pending influence, and usually, we do not need to adjust it.
```

File: pd-control.md [hunk 17]
```diff
@@ -664,50 +683,106 @@ Usage:
...
+    >> scheduler config balance-hot-region-scheduler set min-hot-byte-rate 100
+    ```
+
+- `min-hot-key-rate` means the smallest key counted, which is usually 10.
+
+    ```bash
+    >> scheduler config balance-hot-region-scheduler set min-hot-key-rate 10
+    ```
+
+`max-zombie-rounds` means the maximum number of heartbeats with which an operator can be considered as the pending influence. If you set it to a larger value, more operators might be included in the pending influence. Usually, you do not need to adjust its value. Pending influence refers to the operator influence that is generated during scheduling but still has an effect.
```
Comment (TomShawn) [inline pd-control.md:721]:
```suggestion
- `max-zombie-rounds` means the maximum number of heartbeats with which an operator can be considered as the pending influence. If you set it to a larger value, more operators might be included in the pending influence. Usually, you do not need to adjust its value. Pending influence refers to the operator influence that is generated during scheduling but still has an effect.
```

**Pull request merged**